### DISCUSSION TITLE
frigate SHM oversize

### DISCUSSION THREAD

**Discussion by lukasig** [blakeblackshear/frigate#5985] [Ask A Question] [answered]
sorry if the question is dumb, but what happens if I set a shm size larger than needed for frigate under docker?
Is there any downside to doing this?

**Comment by blakeblackshear** [comment, marked as answer]
It just makes more shared memory available to the container. It could allow frigate to consume more memory than you want. I don't see it being a problem. Addon installs already have half of the available system memory for shm.

**Reply by lukasig** [reply]
Thank you, master frigator ;)

I was thinking this is useful when planing to add one or two extra cameras in the future. Then there would be no problem if I forget to re-run the docker container with the new shm size or if I change the resolution of the detect streams.
